docs/pages/guides/pricing.mdx

```diff
@@ -0,0 +1,130 @@
+---
+meta:
+  title: "Pricing"
+  parentTitle: "Guides"
+  description: "Explanation of Liveblocks 1.0 pricing model and implementation"
+---
+
+With the release of Liveblocks 1.0, we included new pricing plans that are based
+on Monthly Active Users (MAU). This guide will help you understand how to use
+the new pricing plans by providing guidance on the migration to Liveblocks 1.0
+and how to include a `userId` into the `authorize` method if you use it.
+
+## What is a Monthly Active User (MAU)? [#what-is-mau]
+
+In versions before 1.0 we used the number of connections to determine pricing
+segments. We counted each time a user connected to a Liveblocks room from a
+physical device as a connection. We introduced pricing based on Monthly Active
+Users (MAU) to reflect the value that Liveblocks offers to our customers. MAU
+makes pricing more predictable for our users by associating connections to
+users.
+
+A monthly active user can be defined as:
+
+- A `userId` that connects to a Liveblocks room at least once a month (if using
```

Comment:
```suggestion
- A `userId` that connects to a Liveblocks room at least once a month (if using authorization with secret API key)
```

Reply:
@taylorpwhipp When we "using authorization", is it clear enough that it's authorization with the secret API key?
IMO, with the public API key, it's still called authorization.